analysis/report_utils.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for coverage report generation."""
+
+import os
+import multiprocessing
+
+from common import filesystem
+from common import experiment_utils
+from common import new_process
+from common import benchmark_utils
+from common import logs
+from experiment.build import build_utils
+from experiment import measurer
+
+logger = logs.Logger('reporter')  # pylint: disable=invalid-name
+
+
+def get_profdata_file_path(fuzzer, benchmark, trial_id):
```

Comment:
This is all wrong place for this code, analysis only contains analysis and db query code. This should all be in some common/coverage_utils file.

Reply:
I don't think putting it in common makes sense. Why not in experiment?

Reply:
yes experiment seems fine too, just not analysis